posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -731,14 +731,17 @@ public class PostHog private constructor(
         key: String,
         defaultValue: Boolean,
     ): Boolean {
-        if (!isEnabled()) {
-            return defaultValue
+        var value = getFeatureFlag(key, defaultValue)
```

Comment:
you can now delete the `isFeatureEnabled` method from the `remoteConfig` object/class